### PR TITLE
Implement identify tool for rasters

### PR DESCRIPTION
**Pull request by gena**

### CODE REVIEW

File: Map.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 import ee_plugin.utils
 
 
-def addLayer(image: ee.Image, visParams=None, name=None, shown=True, opacity=1.0):
+def addLayer(eeObject, visParams={}, name=None, shown=True, opacity=1.0):
```
Comment (SiggyF) [inline Map.py]: see the warning in the [docs](https://docs.python.org/3/tutorial/controlflow.html#default-argument-values).
In the function arguments it is better to use `..., visParams=None, ...` and then in the function
```
if visParams is None:
        visParams = {}
```

File: ee_plugin.py [hunk 2]
```diff
@@ -13,6 +13,7 @@
 import webbrowser
 from builtins import object
 import os.path
+import json
```
Comment (SiggyF) [inline ee_plugin.py]: See the style guide on [import](https://www.python.org/dev/peps/pep-0008/#imports) order

```
import json
import os.path
import webbrowser

from builtins import object
```
Also note that [`pathlib`](https://docs.python.org/3/library/pathlib.html) provides a more richer api to paths than the older `os.path`

File: utils.py [hunk 3]
```diff
@@ -65,6 +88,50 @@ def get_layer_by_name(name):
...
+        if 'width' in visParams:
+            width = visParams['width']
+
+        color = '000000'
+
+        if 'color' in visParams:
+            color = visParams['color']
+
+        image_fill = features.style(
+            **{'fillColor': color}).updateMask(ee.Image.constant(0.5))
```
Comment (SiggyF) [inline utils.py]: This line might benefit from running [`autopep8`](https://github.com/hhatto/autopep8).

File: utils.py [hunk 4]
```diff
@@ -87,6 +156,20 @@ def add_ee_catalog_image(name, asset_name, visParams, collection_props):
 
     add_or_update_ee_image_layer(image, name)
 
+
 def check_version():
     # check if we have the latest version only once plugin is used, not once it is loaded
     qgis.utils.plugins['ee_plugin'].check_version()
+
+
+def geom_to_geo(geom):
```
Comment (SiggyF) [inline utils.py]: What does this do?

Comment (gena) [reply utils.py]: It converts Qgs geometry to geo, used in provider.py

File: utils.py (hunk 3 above)
Comment (gena) [reply utils.py]: This is what autopep8 has generated

**Pull request merged**